src/main/resources/assets/genetics/lang/zh_CN.lang

```diff
@@ -21,26 +32,39 @@ genetics.item.sequence.3=中等
 genetics.item.sequence.4=强
 genetics.item.sequence.5=极强
 
+genetics.gene.charge.empty=空
+genetics.gene.charge.one=1 次使用
+genetics.gene.charge.many=%d 次使用
+genetics.gene.more=还有 %d 种基因，按下 Shift 显示全部。
+
+genetics.sequence.corrupted=损坏的 DNA 序列
+genetics.sequence.descriptor=%s DNA 序列
+genetics.sequence.unknown=<未知>
+genetics.sequence.unsequenced=未测序
+genetics.sequence.fully=已测序
+genetics.sequence.partially=测序 %d%%
+
 genetics.machine.machine.isolator=基因分离器
-genetics.machine.machine.isolator.info=基因分离器将会从蜜蜂、树苗或者其它的独立个体中随机提取基因片段，并复制到一个空的序列瓶中。此过程需要乙醇作为溶剂，还需要酶来切断DNA。未序列化的血清可以在序列分析器中序列化，并在基因同化器中重新装填。
+genetics.machine.machine.isolator.info=基因分离器将会从蜜蜂、树苗或者其它的独立个体中随机提取基因片段，并复制到一个空的序列瓶中。此过程需要乙醇作为溶剂，还需要酶来切断DNA。未序列化的血清可以在序列分析器中测序，并在基因同化器中重新装填。
 genetics.machine.isolator.error.noIndividual=没有个体样本
-genetics.machine.isolator.error.noRoom=No room for DNA sequences
-genetics.machine.isolator.error.noVials=没有空序列瓶
+genetics.machine.isolator.error.noRoom=没有空间
+genetics.machine.isolator.error.noVials=没有空血清瓶
```

Comment:
Read as "(In the actual GUI,) you need empty DNA sequence, rather than serum vial" - yes I put serum vial in the translation here, but it's midnight for me as I am writing this comment, so just a TODO here.